### DISCUSSION TITLE
How do I use the top and bottom outlet?

### DISCUSSION THREAD

**Discussion by dcj** [TD22057/insteon-mqtt#342] [Q&A] [answered]
I have a number of Insteon outlets (OutletLinc 2663).
These devices have a top and bottom outlet, but both seem to share the same Insteon address.
How to I specify this in the `config.yaml`, and the HA `configuration.yaml`, and the `join` command such that there are two seperate entities?

**Comment by krkeegan** [comment, marked as answer]
https://github.com/TD22057/insteon-mqtt/blob/master/docs/mqtt.md#outlets

**Reply by dcj** [reply]
Thank you!
Yes, this is well documented, but I got tripped up by not scrolling further down in the example config.yaml, I stopped when I found switch, but the outlet section describes how an outlet is really two switches, so this is working well now.
